cmd/promtool/main.go

```diff
@@ -20,8 +20,12 @@ import (
 	"path/filepath"
 	"strings"
 
+	yaml "github.com/ghodss/yaml"
```

Comment:
Changing the rest of our code to also use this new library being something that is far away is a strong reason not to do this.

We do not have consensus on this, and using multiple different libraries within our organisation that do the same thing is to be avoided.